### DISCUSSION TITLE
cannot create user record

### DISCUSSION THREAD

**Discussion by lukepighetti** [pocketbase/pocketbase#5302] [Q&A] [answered]
I'm sure this is a simple mistake on my part but I'm having trouble figuring it out

when calling 

```dart
await _pb.collection("users").create(body: {
      "username": anonUsername, // alphanumeric 16 characters
      "password": anonPassword, // alphanumeric 32 characters
      "passwordConfirm": anonPassword,
    });
```

I get this error 

> ClientException: {url: https://api.streakers.app/api/collections/users/records, isAbort: true, statusCode: 0, response: {}, originalError: ClientException with SocketException: Connection failed (OS Error: Operation not permitted, errno = 1), address = api.streakers.app, port = 443, uri=https://api.streakers.app/api/collections/users/records}

even though the create rule is allow all

my expectation is that it creates a user :) any help is appreciated

note: both `name` and `avatar` values allow empty values, and i have tried explicitly providing them

<img width="706" alt="Screenshot 2024-08-02 at 12 32 00 PM" src="https://github.com/user-attachments/assets/c8284075-5d17-4742-b2cd-b083a3824f68">

<img width="714" alt="Screenshot 2024-08-02 at 12 32 04 PM" src="https://github.com/user-attachments/assets/babc35a5-4799-42e2-bfff-2c964af0c62d">

<img width="721" alt="Screenshot 2024-08-02 at 12 32 14 PM" src="https://github.com/user-attachments/assets/ad21842d-2d57-4fa2-856f-ad226c90b97d">

<img width="940" alt="Screenshot 2024-08-02 at 12 32 46 PM" src="https://github.com/user-attachments/assets/1386a4ae-9d94-4ba1-ab40-5536e50852a9">

**Comment by ganigeorgiev** [comment, marked as answer]
I don't think the error is PocketBase API related. Based on the exception message you could check https://stackoverflow.com/questions/65458903/socketexception-connection-failed-os-error-operation-not-permitted-errno-1 (_see also https://docs.flutter.dev/cookbook/networking/fetch-data#1-add-the-http-package_).

**Reply by lukepighetti** [reply]
configuration issue on my end (macOS outgoing connections), thank you!
